scripts/bootstrap.sh

```
@@ -202,3 +206,9 @@ elif [[ "$MCP_METHOD" == "uv" ]]; then
 else
     echo -e "4. Start the server with: ${BLUE}python -m context_store${NC}"
 fi
+ echo -e "4. Start the server with: ${BLUE}python -m context_store${NC}"
+fi
```

Shellcheck (0.11.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)